docs/rules/multiline-html-element-content-newline.md

```diff
@@ -87,6 +88,8 @@ This rule enforces a line break before and after the contents of a multiline ele
     default `true`
 - `ignores` ... the configuration for element names to ignore line breaks style.  
     default `["pre", "textarea"]`
+- `allowEmptyLine` ... if `true`, allow empty line. If disallow multiple empty lines, use [no-multiple-empty-lines] in combination.  
```

Comment:
Implementation-wise it looks like `allowEmptyLines` might be a bit more correct option 🤔 

As for copy, what do you think about: "... if `true`, it allows empty lines around content. If you want to disallow multiple empty lines, use [...](...)"?

Also, does the core `no-multiple-empty-lines` rule work with Vue template by default?